docs/recorder.md

```diff
@@ -0,0 +1,15 @@
+The recorder works by recording incoming, outgoing messages, the times at which nodes connected/disconnected, times at which itself stopped and started. 
```

Comment:
Please provide a HowTo for the Recorder.
How a QA can use it on a real pool? What steps needs to be done?

Reply:
Please provide this info